packages/webdriverio/tests/scripts/isElementInViewport.test.js

```diff
@@ -0,0 +1,105 @@
+/**
+ * @jest-environment node
```

Comment:
same

Reply:
Should be here, we have it for script tests